### ISSUE TITLE
Update demo.patternlab.io

### ISSUE THREAD

**Issue by bmuenzenmeyer** [pattern-lab/website#122]
People are opening issues due to the current state of http://demo.patternlab.io which is very out of date and will not receive any work - this story exists to update the live demo to something more like https://patternlab-handlebars-preview.netlify.com

**Comment by bmuenzenmeyer** [comment]
@bradfrost can you help with this? I don't think I have access, but the stale state of the demo is [causing issues](https://github.com/pattern-lab/patternlab-node/issues/1091)

**Comment by sghoweri** [comment]
Yes please!! 

@bradfrost lemme know if there’s anything here I can help with!!

**Comment by bradfrost** [comment]
@sghoweri @bmuenzenmeyer I have a page set up locally and am redirecting things here. But I could use some help rounding up and/or creating links to the demos. Thinking of something like this:

## Handlebars

- [Handlebars base starterkit](https://patternlab-handlebars-preview.netlify.com/)
- [Handlebars demo starterkit](https://www.npmjs.com/package/@pattern-lab/starterkit-handlebars-demo)
- [Handlebars vanilla starterkit](https://www.npmjs.com/package/@pattern-lab/starterkit-handlebars-vanilla)

## Twig
- ?

## PL 2 demo
- [Pattern Lab 2 Mustache Demo](http://demo.patternlab.io/)

**Comment by bradfrost** [comment]
@bmuenzenmeyer @sghoweri well I decided to go ahead and publish the page here even though I don't have all the blanks filled in. https://patternlab.io/demos We'll do it live!

**Comment by sghoweri** [comment]
lol I'm good with this. Ship it! 🚀

**Comment by bmuenzenmeyer** [comment]
🚀 

once those links resolve to live demos I suggest we nix http://demo.patternlab.io/

**Comment by bmuenzenmeyer** [comment]
Not sure if we also need to reconcile this content with https://patternlab.io/docs/advanced-starterkits.html - which is more about how to create your own

**Comment by bradfrost** [comment]
@bmuenzenmeyer Yeah we're definitely going to need to zoom out and look at the IA and content of the docs pretty thoroughly. My thought is to do that work as part of the redesign. Which, oh by the way @sghoweri,  I started porting over the site into Eleventy: https://pattern-lab-eleventy.netlify.com/ Only a few hours in, but making progress!

**Comment by hdhdu** [comment]
The roadmap says, that 3.0 ist still beta. But the file is 2 years old. 
Which state is currently the 3.0 version?
Is it still beta or is ist stable?

**Comment by bmuenzenmeyer** [comment]
The site is being rewritten and releases continue past 3.0

https://github.com/pattern-lab/patternlab-node/releases

**Comment by hdhdu** [comment]
Thank you very much for your answer. It is a bit confusing for me, that there is a version 5.xx that contains version 3.0 (not beta). 
I thought, that version 5.xx means the version of patternlab-node which does not have to mean, that patternlab itself is not beta anymore. 
Thanks for rewriting the roadmap page. That will be much more transparent.
